### DISCUSSION TITLE
Help? How to bulk copy Title tag to Album?

### DISCUSSION THREAD

**Discussion by 2600box** [puddletag/puddletag#750] [Q&A]
Hello,

Sorry for asking here, but could not see another way to contact. I cannot find a way to bulk copy Title tags to Album?

Is it possible?

Thanks!

**Comment by sandrotosi** [comment]
can you explain more what you're trying to do? title usually represent the song name, while album is the album's name, and you may not want to copy song's on album's

**Reply by 2600box** [reply]
Indeed, but I am not working with studio albums, but short recorded audio clips.

For the audio library tool I am using they annoyingly _require_ the album field! haha

**Comment by corubba** [comment]
Off the top of my head, I can think of two ways:

Select all the files you want to apply the operation to (probably <kbd>Ctrl</kbd>+<kbd>A</kbd> for all). Open the functions window ("Actions" -> "Functions"), select "Format value" from the dropdown at the top, set "Fields" to `album` and "Format string" to `%title%`. Press the "Ok" button, done.
This way allows you to populate the album field however you want; from constant value, to copied values, to combined and transformed value from multiple other fields.

Select the whole "Title" column by left-clicking the "Title" cell of the first row, press and hold <kbd>Shift</kbd>, and left-click the "Title" cell of the last row. Copy the values by <kbd>Ctrl</kbd>+<kbd>C</kbd>, or right-click and "Copy Selection". Then select the whole "Album" column, then right-click and "Paste Onto Selection".
